src/main/java/lotto/domain/PurchaseResult.java

```diff
@@ -0,0 +1,28 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PurchaseResult {
+
+    private final List<Lotto> purchasedLottos;
+    private final Money payedMoney;
+
+    public PurchaseResult() {
+        this.purchasedLottos = new ArrayList();
+        this.payedMoney = new Money(0);
```

Comment:
똑같은 기능의 객체를 매번 생성하기보다는 객체 하나를 재사용하는 것은 어떨까요?

```suggestion
        this.payedMoney = Money.ZERO;
```